src/index.md

```diff
@@ -1547,7 +1529,66 @@ The Skara tooling includes support for backports. [The official Skara documentat
 [To the top](#){.boxheader}
 :::
 
-# HotSpot development
+# Release Notes
+
+::: {.box}
+[Quick Links]{.boxheader}
+
+* [JDK Release Notes](https://www.oracle.com/java/technologies/javase/jdk-relnotes-index.html)
+* [release-note label description](#release-note)
+:::
+
+Release notes for a product (e.g. the JDK) are part of the release deliverables. They describe changes that are important for a user of the product to know. These are usually things that may affect the user's decision to upgrade to the specific version.
+
+When writing a release note for your feature, be prepared for rather picky review comments about grammar, typos, and wording. This is for the sake of the Java community as a whole, as the language of the release note sets the tone for many blogs and news articles. For a widely used product like the JDK, the release notes are often copied verbatim (including typos) and published to highlight news in the release. This means that we need to take extra care to make sure the text in the release note is correct and has a professional language.
+
+The release note itself is written in a JBS sub-task to the issue that is used to push the change. There are a few steps to follow for the release note to find its way from JBS to the actual release note document.
+
+#. Do these steps for the main issue, that is, the JBS issue that is used to push the original change, **not** on backports or the CSR (if there is one).
+   * Add the label `release-note=yes`.
+   * Create a sub-task.
+#. For the newly created sub-task do these steps:
+   * The **Summary** should be a one sentence synopsis that is informative (and concise) enough to attract the attention of users, developers, and maintainers who might be impacted by the change. The **Summary** should succinctly describe what has actually changed, not be the original bug title, nor describe the problem that was being solved. It should read well as a sub-section heading in a document.
+   * Prefix the **Summary** with "Release Note:".
+   * Add the `release-note` label.
+   * Add the proper `RN-`label (see below).
+   * Add an **Assignee**.
+   * Set the **Fix Version** to the same value that the main issue has.
+   * Enter the text of the release note in the **Description** field using markdown formatting (even though it won't be rendered in JBS).
+#. Ask your Reviewers to have a look at the release note.
+#. When you are done, Resolve the release note sub-task as "Delivered".
+
+For examples of well written release note issues in JBS, see [JDK-8276929](https://bugs.openjdk.java.net/browse/JDK-8276929) or [JDK-8278458](https://bugs.openjdk.java.net/browse/JDK-8278458).
```

Comment:
I think both of these are great in terms of the RN text.  The second, 8278458 for JDK 18, uses label "RN-change" which is deprecated according to lines 1588-1589.  Is this label really deprecated, or just denigrated?